package org.example.Exceptions.SeparateExceptions;

import org.example.CalculatorException;

public class ClassInFactoryNotFoundException extends CalculatorException {
    public ClassInFactoryNotFoundException(){
        super();
    }
}
